cmd/prometheus/main.go

```diff
@@ -150,6 +150,9 @@ func main() {
 	a.Flag("web.console.libraries", "Path to the console library directory.").
 		Default("console_libraries").StringVar(&cfg.web.ConsoleLibrariesPath)
 
+	a.Flag("web.ui-title", "Document title of Prometheus instance.").
```

Comment:
I think I'd call this `web.page-title` or `web.html-title`, with preference towards the former.